src/main/resources/templates/fragments/ppsGlobalErrors.html

```diff
@@ -8,7 +8,7 @@
 <div th:fragment="globalErrors" th:if="${#fields.hasErrors('*')}">
     <div class="govuk-error-summary" role="alert" aria-labelledby="error-summary-heading" tabindex="-1">
         <h2 class="govuk-error-summary__title" id="error-summary-heading">We couldn't find your penalty details</h2>
-        <p id="error-summary-subheading">You must provide a company number and penalty reference to pay a late filing penalty.</p>
+        <p id="error-summary-subheading">You must provide a company number and penalty reference to pay a penalty.</p>
```

Comment:
For now, let's not touch the html pages as they'll be covered when we go through the Confirmation Statement and other journeys. Revert all the template changes and leave them out of this for now.

Reply:
Revert back for the template changes (including the naming of the file part), but one thing is enterPPSDetails is reverted back to enterLFPDetails and the related controller part EnterPPSDetailsController.java (line 51) is also revert back to LFP